elastic/beats
# Add hostname to kube metadata if present

## What does this PR do?

This PR enhances the kube meta to additionally pass hostname if it is set on the pod

## Why is it important?

Additional information that is useful for observability
## Checklist


- [] I have commented my code, particularly in hard-to-understand areas

## Author's Checklist

## How to test this PR locally
Create a pod that sets spec.hostname and ensure that when discovered, hostname is added to pod.hostname.

## Related issues


## Use cases
